var/spack/repos/builtin/packages/dbcsr/package.py

```diff
@@ -4,9 +4,9 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack import *
+from llnl.util.filesystem import working_dir
```

Comment:
I don't think this import is necessary, it should already be available automagically